From bd2b3769684a7f7ebdee1d32c0fe280ef0e75e86 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 21 Oct 2013 15:52:08 -0400
Subject: [PATCH] gtkdoc-mkdb: sort entries in the glossary

The entries in the glossary were in pseudo-random order, varying
even between gtk-doc runs.

https://bugzilla.gnome.org/show_bug.cgi?id=711111

https://bugzilla.gnome.org/show_bug.cgi?id=711112
---
 gtkdoc-mkdb.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtkdoc-mkdb.in b/gtkdoc-mkdb.in
index e448ed7..faffaf4 100755
--- a/gtkdoc-mkdb.in
+++ b/gtkdoc-mkdb.in
@@ -1119,7 +1119,7 @@ $header
   <title>Annotation Glossary</title>
 EOF
 
-    foreach my $annotation (keys(%AnnotationsUsed)) {
+    foreach my $annotation (sort(keys(%AnnotationsUsed))) {
         if(defined($AnnotationDefinition{$annotation})) {
             my $def = $AnnotationDefinition{$annotation};
             my $curletter = uc(substr($annotation,0,1));
-- 
1.8.3.1

